src/mrinufft/trajectories/display.py

```diff
@@ -18,17 +24,22 @@
 ]
 NB_COLORS = len(COLOR_CYCLE)
 
-
+ALPHA = 0.2
 LINEWIDTH = 2
 POINTSIZE = 10
 FONTSIZE = 18
+SMALL_FONTSIZE = 14
```

Comment:
Those global values are fine but I think we can do better, especially if the user wants to modify them, their change would not be taken into account in the parent module when imported. 
Something like the matplotlib `rcParams` config might be desirable.

Reply:
A singleton has been implemented and integrated in all display functions to enable runtime user configuration